Public/Src/Engine/Dll/EngineSchedule.cs

```diff
@@ -211,20 +211,25 @@ public static PipTable CreateEmptyPipTable(PipExecutionContext context)
                 graphSemistableFingerprint: pipGraph.SemistableFingerprint,
                 environmentFingerprint: configuration.Schedule.EnvironmentFingerprint);
 
-            AsyncLazy<PipRuntimeTimeTable> runtimeTable = Lazy.CreateAsync(() => TryLoadRunningTimeTable(
+            Task<PipRuntimeTimeTable> runtimeTableTask = TryLoadRunningTimeTable(

```

Comment:
>+  [](start = 63, length = 2)

May be add ':' as a separator.

Reply:
I don't remember that I have ever changed this line:214 EngineSchedule.cs